File: Experiment1.cpp
#include <iostream>
using namespace std;

int main() {

    int i, a[5], b[5], c[5];
    int ch, sum = 0, sub = 0;
    char x;

    do {
        cout << "\n1. Create" << endl;
        cout << "2. Display" << endl;
        cout << "3. Sum" << endl;
        cout << "4. Substraction" << endl;
        cout << "Enter your choice: ";
        cin >> ch;

        switch(ch) {
        case 1:
            cout << "Enter 5 elements for array a: ";
            for(i = 0; i < 5; i++)
                cin >> a[i];
            break;

        case 2:
            cout << "Elements in array a are: ";
            for(i = 0; i < 5; i++)
                cout << a[i] << " ";
            cout << endl;
            break;

        case 3:
            sum = 0;
            for(i = 0; i < 5; i++)
                sum += a[i];
            cout << "Sum = " << sum << endl;
            break;

        case 4:
            cout << "Enter 5 elements for array b: ";
            for(i = 0; i < 5; i++)
                cin >> b[i];
            cout << "Subtraction (a - b) = ";
            for(i = 0; i < 5; i++) {
                c[i] = a[i] - b[i];
                cout << c[i] << " ";
            }
            cout << endl;
            break;

        default:
            cout << "Invalid choice" << endl;
        }

        cout << "Continue? (y/n): ";
        cin >> x;

    } while(x == 'y' || x == 'Y');

    return 0;
}

File: Experiment1.java
import java.util.*;

public class Exp1 {

    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter size of array: ");
        int n = sc.nextInt();

        int a[] = new int[n];
        int b[] = new int[n];
        int c[] = new int[n];

        int choice;

        do {
            System.out.println("\n\n----MENU----");
            System.out.println("1. Create A");
            System.out.println("2. Create B");
            System.out.println("3. Display A");
            System.out.println("4. Display B");
            System.out.println("5. Addition");
            System.out.println("6. Subtraction");
            System.out.println("7. Multiplication");
            System.out.println("8. Division");
            System.out.println("9. Exit");

            System.out.print("Enter your choice: ");
            choice = sc.nextInt();

            switch (choice) {

                case 1:
                    System.out.print("Enter elements of A: ");
                    for (int i = 0; i < n; i++) {
                        a[i] = sc.nextInt();
                    }
                    break;

                case 2:
                    System.out.print("Enter elements of B: ");
                    for (int i = 0; i < n; i++) {
                        b[i] = sc.nextInt();
                    }
                    break;

                case 3:
                    System.out.print("Array A: ");
                    for (int i = 0; i < n; i++) {
                        System.out.print(a[i] + " ");
                    }
                    break;

                case 4:
                    System.out.print("Array B: ");
                    for (int i = 0; i < n; i++) {
                        System.out.print(b[i] + " ");
                    }
                    break;

                case 5:
                    System.out.print("Addition: ");
                    for (int i = 0; i < n; i++) {
                        c[i] = a[i] + b[i];
                        System.out.print(c[i] + " ");
                    }
                    break;

                case 6:
                    System.out.print("Subtraction: ");
                    for (int i = 0; i < n; i++) {
                        c[i] = a[i] - b[i];
                        System.out.print(c[i] + " ");
                    }
                    break;

                case 7:
                    System.out.print("Multiplication: ");
                    for (int i = 0; i < n; i++) {
                        c[i] = a[i] * b[i];
                        System.out.print(c[i] + " ");
                    }
                    break;

                case 8:
                    System.out.print("Division: ");
                    for (int i = 0; i < n; i++) {
                        c[i] = a[i] / b[i];
                        System.out.print(c[i] + " ");
                    }
                    break;

                case 9:
                    System.out.println("Exit...");
                    break;

                default:
                    System.out.println("Invalid choice!");
            }

        } while (choice != 9);

    }
}
